transaction-view/src/bytes.rs

```diff
@@ -32,6 +32,22 @@ pub fn read_byte(bytes: &[u8], offset: &mut usize) -> Result<u8> {
     value
 }
 
+/// Read a byte and advance the offset without any bounds checks.
+///
+/// * `bytes` - Slice of bytes to read from.
+/// * `offset` - Curernt offset into `bytes`.
+/// * `num_elements` - Number of `T` elements in the slice.
```

Comment:
```suggestion
/// * `offset` - Current offset into `bytes`.
```
typo and unnecessary line

Reply:
Had same typo elsewher: 9e8016a8a0